### PR TITLE
Change documenting of Docker version

### PR DESCRIPTION
**Pull request by m-minasyan**
1. Change documenting of `Docker` version to `Docker-Moby` version in `docker-moby.sh` for Ubuntu 16 and Ubuntu 18.
2. Delete duplicate of documenting `Docker` version.

### CODE REVIEW

**Comment by miketimofeev** [comment]
Do we really need both docker.sh and docker-moby.sh?

File: images/linux/scripts/installers/docker.sh [hunk 1]
```diff
@@ -26,9 +26,4 @@ docker pull debian:9
 docker pull alpine:3.7
 docker pull alpine:3.8
 docker pull alpine:3.9
-docker pull alpine:3.10
-
-## Add version information to the metadata file
-echo "Documenting Docker version"
-DOCKER_VERSION=`docker -v`
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/docker.sh]: Just wonder, why we delete these lines.
Previously, `Docker` doc entry was produced in 2 places:
1) docker-moby (that you fixed)
2) docker (that you have removed)
So where the original `Docker` is installed? Or we don't install original docker at all?

Comment (m-minasyan) [reply images/linux/scripts/installers/docker.sh]: Its shows same thing.
https://github.com/actions/virtual-environments/releases/tag/ubuntu18%2F20200217.1

**Comment by m-minasyan** [comment]
> Do we really need both docker.sh and docker-moby.sh?

I think we need only one of it, because its do same things.

**Comment by al-cheb** [comment]
@m-minasyan Please merge latest changes to your branch from the master branch.

File: images/linux/scripts/installers/docker.sh [hunk 2]
```diff
@@ -1,32 +0,0 @@
...
-##  File:  docker.sh
-##  Desc:  Installs the correct version of docker onto the image, and pulls
-##         down the default docker image used for building on ubuntu
-################################################################################
-
-source $HELPER_SCRIPTS/apt.sh
-source $HELPER_SCRIPTS/document.sh
-
-DOCKER_PACKAGE=moby
-
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/docker.sh]: Should we also remove `docker.sh` from packer templates?

**Comment by maxim-lobanov** [comment]
Fix for the issue #290

**Pull request merged**